src/display/editor/tools.js

```diff
@@ -61,9 +61,13 @@ class IdManager {
    * Get a unique id.
    * @returns {string}
    */
-  getId() {
+  get id() {
     return `${AnnotationEditorPrefix}${this.#id++}`;
   }
+
+  reset() {
+    this.#id = 0;
+  }
```

Comment:
This doesn't seem like a method we should be exposing anywhere *except* when running tests. Please move this to the constructor instead:
```javascript
if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
  Object.defineProperty(this, "reset", {
    value: () => {
      this.#id = 0;
    },
  });
}
```